js/views/ListingCard.js

```diff
@@ -265,6 +295,25 @@ export default class extends baseVw {
     this._$btnEdit = null;
     this._$btnDelete = null;
 
+    const imageLoaderKey = `_${this.viewType}ViewListingImageLoad`;
+
+    if (this[imageLoaderKey]) {
+      this[imageLoaderKey].always(imgUrl => {
+        let url = 'url("../imgs/defaultItem.png")';
+
+        // there will not be an imgUrl if the listing image failed to load
+        if (imgUrl) {
+          url = `url("${imgUrl}"), ${url}`;
+        }
+
+        this.getCachedEl(`.js-${this.viewType}ViewListingImage`)[0]
+          .style
+          .backgroundImage = url;
+        this.getCachedEl('.js-listingImageLoadSpinner')
```

Comment:
Based on some experiments I did along with some thoughts in [this thread](https://stackoverflow.com/questions/46333377/will-hidden-obscured-svg-animations-still-cause-browser-repaints-and-or-perfor), I'm removing the spinner svg once the image loads. Even if it is obscured or hidden, it's taking up resources.